### ISSUE TITLE
[Package Request] - aws-cfn-bootstrap

### ISSUE THREAD

**Issue by msamsonoff** [amazonlinux/amazon-linux-2022#46]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**
`aws-cfn-bootstrap` for configuring instances with CloudFormation.

**Is this an update to existing package or new package request?**
New package.

**Is this package available in Amazon Linux 2?**
Yes.

**Comment by aviorma** [comment]
how can I install it manually?

**Comment by mello7tre** [comment]
This is my sample user-data can work for both amazonlinux2 and amazolinux2022.
If it was not able to install package from repo it download and install it by `source`.
```
#!/bin/bash
PATH=/opt/aws/bin:/usr/local/bin:$PATH
yum -C list installed aws-cfn-bootstrap || yum install -y aws-cfn-bootstrap
if ( ! which cfn-init );then
  cd /opt
  curl https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz | tar zx
  cd aws-cfn-bootstrap-2.0
  dnf install -y python python-pip3 chkconfig
  pip3 install . && cp init/systemd/cfn-hup.service /etc/systemd/system/ && systemctl daemon-reload
fi
```

**Comment by stewartsmith** [comment]
Just a quick update on this: we have been actively working with the CloudFormation team on this, which mostly goes down to discussing dependencies and how we be very intentional what we bring into the OS for the next 5 years. It does indeed look like we have a way forward to have a package out shortly.

**Comment by stewartsmith** [comment]
The `aws-cfn-bootstrap` package is available at least as of the `2022.0.20220728` release.
